### DISCUSSION TITLE
fetch a certain number of items from base

### DISCUSSION THREAD

**Discussion by ooralas** [deta/cloud-docs#278] [Help] [answered]
is it somehow possible that I get with the "get Request" only a certain number of entries, e.g. I have in my database 150 entries and I want from it only 15 entries, I know that I can get all entries and then take 15 of them, but it is important to me here that the request would be short.

**Comment by jnsougata** [comment]
Deta base post  `/query`  has `limit` parameter with that you can control number of entries to be fetched. you can try that.
Reference : https://docs.deta.sh/docs/base/http#query-items

**Reply by ooralas** [reply]
thanks for your answer.
I need that in a get-Request not in post :).

**Comment by ooralas** [comment, marked as answer]
By the way i found the solution using `fetch`. Somehow I overlooked fetch. There you can enter in optional `limit `parameter.
Reference : https://docs.deta.sh/docs/base/sdk#fetch)

**Reply by jnsougata** [reply]
`fetch` in all sdk uses `post` request under the hood. They actually uses the `query` endpoint with is done through POST. If you are thinking that you are doing `get-Request` by using SDK well then you are wrong as per your requirement.

**Reply by ooralas** [reply]
ah okay i didn't know that, thanks for the info :)
